pkgs/by-name/fr/freefilesync/skip-missing-Animal.dat.patch

```diff
@@ -0,0 +1,66 @@
+index a40eea3..28c5b0c 100644
+--- a/FreeFileSync/Source/ui/gui_generated.cpp
++++ b/FreeFileSync/Source/ui/gui_generated.cpp
+@@ -5573,8 +5573,8 @@ AboutDlgGenerated::AboutDlgGenerated( wxWindow* parent, wxWindowID id, const wxS
+ 	wxBoxSizer* bSizer183;
+ 	bSizer183 = new wxBoxSizer( wxHORIZONTAL );
+ 
+-	m_bitmapAnimalSmall = new wxStaticBitmap( m_panelDonate, wxID_ANY, wxNullBitmap, wxDefaultPosition, wxDefaultSize, 0 );
+-	bSizer183->Add( m_bitmapAnimalSmall, 0, wxALIGN_CENTER_VERTICAL|wxALL, 5 );
++	// m_bitmapAnimalSmall = new wxStaticBitmap( m_panelDonate, wxID_ANY, wxNullBitmap, wxDefaultPosition, wxDefaultSize, 0 );
++	// bSizer183->Add( m_bitmapAnimalSmall, 0, wxALIGN_CENTER_VERTICAL|wxALL, 5 );
+ 
+ 	wxPanel* m_panel39;
+ 	m_panel39 = new wxPanel( m_panelDonate, wxID_ANY, wxDefaultPosition, wxDefaultSize, wxTAB_TRAVERSAL );
+@@ -5587,13 +5587,13 @@ AboutDlgGenerated::AboutDlgGenerated( wxWindow* parent, wxWindowID id, const wxS
+ 	m_staticTextDonate->Wrap( -1 );
+ 	m_staticTextDonate->SetForegroundColour( wxColour( 0, 0, 0 ) );
+ 
+-	bSizer184->Add( m_staticTextDonate, 0, wxALIGN_CENTER_HORIZONTAL|wxLEFT|wxALIGN_CENTER_VERTICAL, 10 );
++	// bSizer184->Add( m_staticTextDonate, 0, wxALIGN_CENTER_HORIZONTAL|wxLEFT|wxALIGN_CENTER_VERTICAL, 10 );
+ 
+ 
+ 	m_panel39->SetSizer( bSizer184 );
+ 	m_panel39->Layout();
+ 	bSizer184->Fit( m_panel39 );
+-	bSizer183->Add( m_panel39, 1, wxTOP|wxBOTTOM|wxRIGHT|wxEXPAND, 5 );
++	// bSizer183->Add( m_panel39, 1, wxTOP|wxBOTTOM|wxRIGHT|wxEXPAND, 5 );
+ 
+ 
+ 	m_panelDonate->SetSizer( bSizer183 );
+diff --git a/FreeFileSync/Source/ui/small_dlgs.cpp b/FreeFileSync/Source/ui/small_dlgs.cpp
+index 933fe81..b79661b 100644
+--- a/FreeFileSync/Source/ui/small_dlgs.cpp
++++ b/FreeFileSync/Source/ui/small_dlgs.cpp
+@@ -137,9 +137,9 @@ AboutDlg::AboutDlg(wxWindow* parent) : AboutDlgGenerated(parent)
+ 
+     wxImage::AddHandler(new wxJPEGHandler /*ownership passed*/); //activate support for .jpg files
+ 
+-    wxImage animalImg(utfTo<wxString>(appendPath(getResourceDirPath(), Zstr("Animal.dat"))), wxBITMAP_TYPE_JPEG);
+-    convertToVanillaImage(animalImg);
+-    assert(animalImg.IsOk());
++    // wxImage animalImg(utfTo<wxString>(appendPath(getResourceDirPath(), Zstr("Animal.dat"))), wxBITMAP_TYPE_JPEG);
++    // convertToVanillaImage(animalImg);
++    // assert(animalImg.IsOk());
+ 
+     //--------------------------------------------------------------------------
+     //have animal + text match *final* dialog width
+@@ -150,13 +150,13 @@ AboutDlg::AboutDlg(wxWindow* parent) : AboutDlgGenerated(parent)
+ #endif
+ 
+     {
+-        const int imageWidth = (m_panelDonate->GetSize().GetWidth() - 5 - 5 - 5 /* grey border*/) / 2;
+-        const int textWidth  =  m_panelDonate->GetSize().GetWidth() - 5 - 5 - 5 - imageWidth;
++        // const int imageWidth = (m_panelDonate->GetSize().GetWidth() - 5 - 5 - 5 /* grey border*/) / 2;
```

Comment:
Please delete code to have a cleaner patch file

Reply:
Done.